src/server/lib/user-features.ts

```diff
@@ -29,6 +36,17 @@ export const setUserFeatureCookies = async ({
 		accessToken,
```

Comment:
Not related to this line, but can you update the comment/description of this function `setUserFeatureCookies` to include we're also pulling from the user-benefits-api now?